modules/cluster-logging-elasticsearch-exposing.adoc

```diff
@@ -152,7 +152,7 @@ $ routeES=`oc get route elasticsearch -o jsonpath={.spec.host}`
 +
 [source,terminal]
 ----
-$ curl -tlsv1.2 --insecure -H "Authorization: Bearer ${token}" "https://${routeES}/.operations.*/_search?size=1" | jq
+$ curl -tlsv1.2 --insecure -H "Authorization: Bearer ${token}" "https://${routeES}/infra-*/_search?size=1" | jq
```

Comment:
How about use the command below?  use jq is not a good idea.
curl -tlsv1.2 --insecure -H "Authorization: Bearer ${token}" "https://${routeES}"
{
  "name" : "elasticsearch-cdm-i40ktba0-1",
  "cluster_name" : "elasticsearch",
  "cluster_uuid" : "0eY-tJzcR3KOdpgeMJo-MQ",
  "version" : {
    "number" : "6.8.1",
    "build_flavor" : "oss",
    "build_type" : "zip",
    "build_hash" : "Unknown",
    "build_date" : "Unknown",
    "build_snapshot" : true,
    "lucene_version" : "7.7.0",
    "minimum_wire_compatibility_version" : "5.6.0",
    "minimum_index_compatibility_version" : "5.0.0"
  },
  "tagline" : "You Know, for Search"
}

Reply:
Excellent - thanks.

Reply:
@anpingli  Sorry. The formatting is off. Would you resend this using markdown code block (between triple-backticks, like this ``` )?

Reply:
@anpingli Please have a look at the updated command and example output. To get the correct formatting, would it be possible to provide the example output between triple backticks (```) like this?
``` 
backticks 
```